definitions/base_classes/NXdetector.nxdl.xml

```diff
@@ -817,4 +817,19 @@
           for a summary of the discussion.
       </doc>
   </attribute>
+  <field name="depends_on" type="NX_CHAR">
+    <doc>
+      Points to the path to a field defining the last transformation in a chain of transformations
+      which defines the position and orientation of this component or the string "." if located
+      in the origin.
+
+      The reference point of the detector is the center of the first pixel.
+      In complex geometries the NXoff_geometry groups can be used to provide an umabigous reference.
```

Comment:
umabigous -> unambiguous?

Reply:
```suggestion
      In complex geometries the NXoff_geometry groups can be used to provide an unambiguous reference.
```
There is a beautiful "insert a suggestion" functionality in GitHub. Feel free to use it.